test/unit/application.spec.js

```diff
@@ -65,20 +65,6 @@ describe('marionette application', function() {
     });
   });
 
-  describe('when an app has been started, and registering another initializer', function() {
```

Comment:
We haven't quite agreed on what the api should look like, but it'd be something like this:


```js
var App = Application.extend({
  initialize: function() {
    console.log('app!');
    this.router = new App.Router();
  }
});

App.Router = Router.extend({
  routes: {
    'colors' : function() { return new ColorsApp(); }
  }
});

var ColorsApp = Application.extend({
  initialize: function() {
    console.log('colors!');
  }
});

Backbone.history.start();
var app = new App();
// app!
app.router.navigate('colors');
// colors!
```